lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
On second thought, this approach is flawed because any code in the background thread called after `FusedLocationProviderApi#requestLocationUpdates` will not be executed. Maybe we can simplify the solution to this problem and just run the call to register for location updates with the `LocationManager` on the main thread if we detect there is no looper in the calling thread. Can you think of any downside to this? Also, after further testing, it appears that other libraries don't create a looper for a given thread if there isn't one.